docs/_sources/vc_features.rst.txt

```diff
@@ -0,0 +1,36 @@
+VC features
+===========
+Conda builds Windows packages with features as a means of keeping the Visual Studio version (``vc``) used to build packages uniform across a given Python version.
+
+Rule of thumb
+-------------
+* if it's a shared library that something other than Python will use, you only need python in build to activate features. You need the features block;
```

Comment:
Can you mention here that either py35 or py36 needs to be disabled as they both use the same MSVC version?